### ISSUE TITLE
Cannot bake partially filled pies

### ISSUE THREAD

**Issue by Ziver7** [anegostudios/VintageStory-Issues#1346]
**Game Version:** 1.15.6
**Platform:** Windows 
**Modded:** Yes

- Alpha Weapons Pack
- Armor Crafting Rebalance
- Better-Drifters
- CarryCapacity
- VSProspector Info

### Description
Cannot bake a partially filled pie in an oven. I receive a "Item cannot be baked" message

### How to reproduce
1. Make an empty pie crust on a table. (I used Rye dough)
2. Place two fillings (I used 2 carrots and 2 onion)
3. Try to place raw, partially filled pie into oven
4. Get error message

**Comment by copygirl** [comment]
Yes, I believe this is intentional. The other issue should probably be closed, too.
